pkg/eventlog/event.go

```diff
@@ -8,6 +8,9 @@ import (
 type Event struct {
 	// ID is identifier, also used as a pointer reference target.
 	ID string `json:"_id" bson:"_id,omitempty"`
+	// Op is the operation on the event log. Valid values are "ADD"
+	// TODO: add support for "DEL"
+	Op string `json:"op" bson:"op"`
```

Comment:
Let's have a discussion about DEL, i have feedback from people that they would like to keep modules present even though they were deleted in VCS